source/vistas/core/graphics/renderable.py

```diff
@@ -43,12 +43,28 @@ def __init__(self):
         self.scale = Vector3([1, 1, 1])
         self.position = Vector3()
         self.rotation = Vector3()
+        self._bounding_box = None
+
+        self.bbox_vao = -1
+        self.bbox_vertex_buffer = -1
+        self.bbox_index_buffer = -1
+
+        # Init the VAO
         self.bounding_box = BoundingBox(0, 0, 0, 0, 0, 0)
 
-    def render(self, camera):
-        pass
+    def __del__(self):
+        if self.bbox_vao != -1:
+            glDeleteVertexArrays(1, [self.bbox_vao])
```

Comment:
The second argument for these do not need to be lists. `glDeleteVertexArrays(1, self.bbox_vao)` works just fine.